web/styles/popovers.css

```diff
@@ -671,6 +671,8 @@ ul.popover-group-menu-member-list {
 
                     &:first-of-type {
```

Comment:
It looks like there's a class, `.subscription_list_stream`, which could be used here instead of `:first-of-type`. Likewise with the `:last-of-type`, which can be referenced via another class. That could make a good prep commit for this PR.